requirements.in

```diff
@@ -2,6 +2,7 @@ click==7.0
 dj-database-url==0.5.0
 django-admin-sortable2
 django-autoslug==1.9.4
+django-bleach==3.0.0
```

Comment:
I'd probably avoid pinning bleach here. I don't update that package drastically really at all - it's a stable app now so just gets django support updates occasionally.

But if we do want it pinned, maybe make it `<4`. I've not no plans to move to v4 soon as v3 supports django 3.2 through to 4.1

Reply:
Yeah that works, but I'd always use `<4` because it's more common than using `~=`. Just to avoid confusion for anyone reading the requirements